cmd/limactl/clone.go

```diff
@@ -55,7 +71,7 @@ func cloneAction(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	newInst, err := instance.Clone(ctx, oldInst, newInstName)
+	newInst, err := instance.CloneOrRename(ctx, oldInst, newInstName, rename)
```

Comment:
I expected `rename` to just rename the instance directory and instance name, not to clone and delete the original.

This is potentially an issue when you have no copy-on-write filestem, so you temporarily need twice the amount of disk space. It might also be really slow on Windows when you have to export the rootfs as a tarball and reimport it as a different distro.

But I guess we can start with this as the initial implementation and improve it later.